src/components/Reown/SendTransactionModal.js

```diff
@@ -0,0 +1,439 @@
+      return '-';
+    }
+
+    return numberUtils.prettyValue(value);
+  };
+
+  const truncateTxId = (txId) => {
+    if (!txId) return '-';
+    return `${txId.slice(0, 12)}...${txId.slice(-12)}`;
+  };
+
+  const copyToClipboard = (text) => {
+    Clipboard.setString(text);
+  };
+
+  const renderInputs = () => {
+    if (!data?.data?.inputs || data.data.inputs.length === 0) {
+      return null;
+    }
+
+    return (
+      <View>
+        <Text style={styles.sectionTitle}>{t`Inputs`}</Text>
+        {data.data.inputs.map((input, index) => {
+          const inputKey = `input-${input.txId || ''}-${input.index || ''}-${index}`;
+          return (
+            <View
+              key={inputKey}
+              style={styles.itemContainer}
+            >
+              <View style={styles.itemHeader}>
+                <Text style={styles.itemTitle}>{t`Input ${index + 1}`}</Text>
+                <Text style={styles.valueText}>
+                  {formatValue(input?.value)} {getTokenSymbol(input?.token)}
+                </Text>
+              </View>
+              <Text style={styles.labelText}>{t`ID da Transação`}</Text>
+              <View style={styles.valueContainer}>
+                <Text style={styles.monospace}>
+                  {truncateTxId(input?.txId)} ({input?.index})
+                </Text>
+                {input?.txId && (
+                  <TouchableOpacity onPress={() => copyToClipboard(input.txId)}>
+                    <Text style={styles.copyButton}>{t`Copy ID`}</Text>
+                  </TouchableOpacity>
+                )}
+              </View>
+              <Text style={styles.labelText}>{t`Endereço`}</Text>
+              <TouchableOpacity
+                style={styles.addressContainer}
+                onPress={() => copyToClipboard(input?.address)}
+              >
+                <Text style={styles.monospace} numberOfLines={1}>
+                  {input?.address}
+                </Text>
+              </TouchableOpacity>
+            </View>
+          );
+        })}
+      </View>
+    );
+  };
+
+  const renderOutputs = () => {
+    if (!data?.data?.outputs || data.data.outputs.length === 0) {
+      return null;
+    }
+
+    return (
+      <View>
+        <Text style={styles.sectionTitle}>{t`Outputs`}</Text>
+        {data.data.outputs.map((output, index) => {
+          const outputKey = `output-${output.address || ''}-${output.token || ''}-${Math.random().toString(36).substr(2, 9)}`;
+          return (
+            <View
+              key={outputKey}
+              style={styles.itemContainer}
+            >
+              <View style={styles.itemHeader}>
+                <Text style={styles.itemTitle}>{t`Output ${index + 1}`}</Text>
+                <Text style={styles.valueText}>
+                  {formatValue(output?.value)} {getTokenSymbol(output?.token)}
+                </Text>
+              </View>
+              <Text style={styles.labelText}>{t`Endereço`}</Text>
```

Comment:
```suggestion
              <Text style={styles.labelText}>{t`Address`}</Text>
```

Reply:
Fixed, thanks

The fix is in SendTransactionRequest component